docs/release.md

```diff
@@ -0,0 +1,55 @@
+# Status AddOn Relesea process 
+
+The Status AddOn release process is based on [GoReleaser](https://goreleaser.com).
```

Comment:
A GitHub workflow also plays an important part. Also, I would add a very brief high-level characterization of what these things accomplish. I might reword this sentence something like the following.

> This GitHub repository has an automated process that creates a release corresponding to each Git tag whose name starts with "v".

Reply:
ok, updated